guides/plugins/apps/starter/starter-admin-extension.md

```diff
@@ -0,0 +1,155 @@
+
+{% hint style="info" %}
+When you are using a self-managed Shopware Version, make sure to set the feature flag `FEATURE_NEXT_17950=1` to enable the Admin Extension API.
+{% endhint %}
+
+![An admin notification](../../../../.gitbook/assets/extension-api-notification.png)
+
+## Prerequisites
+
+In order to follow this guide, make sure you're familiar with and meet the following requirements:
+
+ * Basic CLI usage (creating files, directories, running commands)
+ * Installed [shopware-cli](https://sw-cli.fos.gg/) tools
+ * We will use the following libraries / softwares
+    * npm
+    * live-server (small local development live-reloading server)
+
+## Create the App Wrapper
+
+First of all we need to create the app "wrapper", the so-called app manifest. It is just a single XML file with some basic configuration.
+
+### Create manifest file
+
+First of all, we create a new directory that contains our project.
+
+```bash
+mkdir ListingExtension
+```
+
+within that directory, we create the manifest file.
+
+```bash
+cd ListingExtension
+touch manifest.xml
+```
+
+{% hint style="info" %}
+When you are using a self-managed Shopware Version, you can also create the app base directory in the `custom/apps` directory of your Shopware installation. However, the descriptions in this guide apply to both Shopware cloud and self-managed stores.
+{% endhint %}
+
+Next, we're gonna put our basic configuration into the file we just created.
+
+{% code title="manifest.xml" %}
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+        <name>ListingExtension</name>
+        <label>Listing Extension App</label>
+        <description>This app extends the product listing</description>
+        <author>shopware AG</author>
+        <copyright>(c) shopware AG</copyright>
+        <version>1.0.0</version>
+        <license>MIT</license>
+    </meta>
+</manifest>
+
+```
+{% endcode %}
+
+## Set up communication between Shopware and the App
+
+Next, we need to set up an entry point, so Shopware and your app can communicate. The entry point is a static `.html` file, which includes the Extension SDK script and defines our extension.
+
+![Communication between the admin panel and your entry point](../../../../.gitbook/assets/extension-api-communication.png)
+
+The file will be rendered as a hidden iFrame within your admin panel. Using `postMessage` requests, the iFrame and your admin panel can communicate and exchange data.
+
+Let's create this file in a directory called `src`.
+
+```bash
+mkdir src
+touch src/index.html
+```
+
+{% code title="index.html" %}
+```html
+<!doctype html>
+<html>
+    <head>
+        <script src="https://unpkg.com/@shopware-ag/admin-extension-sdk/cdn"></script>
```

Comment:
Should we pin here to a specific version? So this doc does not break when we break the API?